### ISSUE TITLE
[UI] Views with searchbar are not mobile friendly

### ISSUE THREAD

**Issue by Rolikasi** [taskcluster/taskcluster#1670]
**To Reproduce**
1. Go to http://localhost:5080/tasks 
2. Make the width of the screen smaller

**Actual**
![Mobile issue tasks](https://user-images.githubusercontent.com/44370635/67003097-128bf680-f0ff-11e9-9ce6-1996d464a8c9.PNG)

**How it looks on wider screen**
![Mobile issue tasks 2](https://user-images.githubusercontent.com/44370635/67003196-45ce8580-f0ff-11e9-85dd-9a0b48bbf6cd.PNG)

**Additional context**
The same issue for:
http://localhost:5080/tasks/groups
http://localhost:5080/auth/clients
http://localhost:5080/auth/roles
http://localhost:5080/auth/scopes
http://localhost:5080/worker-manager
http://localhost:5080/hooks
http://localhost:5080/secrets
http://localhost:5080/notify/denylist

**Comment by helfi92** [comment]
https://github.com/taskcluster/taskcluster/pull/1596 should hopefully fix this issue.

**Comment by Soundharyaam** [comment]
@helfi92 Yup. I am working on it. Two tasks are pending on my todos. Once it's done, I will let you know. It's a fix for almost all the pages which has entities such as `Search bar`, and a link for  `Documentation`, `Account`, `Sign-in/Sign-out` routes.

**Comment by SharifAly** [comment]
hy i want to ask, need you help on this issue or it's fixed, i am new in open source, and i want find a project to work on it ??

**Comment by Priyanko9** [comment]
Is the issue is still there,Can any one assigned it to me ?

**Comment by lotas** [comment]
hi @Priyanko9, this is still a case. Feel free to grab this one. 
Thanks!
